specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -226,10 +226,10 @@ paths:
             type: string
         - in: query
           name: typegerechtigde
-          description: "Een typering van het recht dat de zakelijkgerechtigde heeft op de Kadastraal Onroerende Zaak. De mogelijke waarden van deze typering is te vinden in een [waardelijst](http://www.kadaster.nl/schemas/waardelijsten/AardZakelijkRecht).  "
+          description: "Een typering van het recht dat de zakelijkgerechtigde heeft op de Kadastraal Onroerende Zaak."
           required: false
           schema:
-            type: string
+            $ref: "#/components/schemas/typeGerechtigde_enum"
```

Comment:
Moet dit niet op dezelfde manier opgelost worden als voor andere waardenlijsten? Dus hier de code meegeven van het typegerechtigde. Met de waardenlijsten API kan de code opgevraagd worden van bijvoorbeeld het type 'eigenaar'.

Reply:
Dat kunnen we doen als dit ook een formle waardelijst is die gepubliceerd wordt door het Kadaster. Vooralsnog is het een lijst die specidfiek voor deze API verzonnen is en alleen toegepast wordt voor gemeenten . Een enumeratie is daarvoor vooralsnog de beste oplossing.